src/oatmeal-pancakes.md

```diff
@@ -0,0 +1,30 @@
+# Oatmeal Pancakes
+![oatmeal-pancakes](pix/oatmeal-pancakes.webp)
+
+- ⏲️ Prep time: 10 min
+- 🍳 Cook time: 15 min
+- 🍽️ Servings: 8
+
+## Ingredients
+
+- 2 eggs
+- 40g sugar (3 tbsp)
+- 20g honey (1 tbsp)
+- 60ml milk (4 tbsp) (or you can use water instead)
```

Comment:
It would be more concise to say, `you can use water instead and leave out the `or`.